drivers/SmartThings/harman-luxury/profiles/maX10.yaml

```diff
@@ -23,7 +23,7 @@ components:
   - id: refresh
     version: 1
   categories:
-  - name: Speaker
+  - name: Receiver
```

Comment:
This is the change.

Reply:
Oh I forgot that this PR is grubbing all my pushed commits.
This was actually part of a new commit - PR #1357 

So I will close this PR now